tests/monster_test_generated.js

```diff
@@ -0,0 +1,936 @@
+// automatically generated by the FlatBuffers compiler, do not modify
+
+/**
+ * @const
+*/
+var MyGame = MyGame || {};
+
+/**
+ * @const
+*/
+MyGame.Example = MyGame.Example || {};
+
+/**
+ * @const
+*/
+MyGame.OtherNameSpace = MyGame.OtherNameSpace || {};
+
+/**
+ * @enum
+ */
+MyGame.Example.Color = {
+  Red: 1,
+  Green: 2,
+  Blue: 8
+};
+
+/**
+ * @enum
+ */
+MyGame.Example.Any = {
+  NONE: 0,
+  Monster: 1,
+  TestSimpleTableWithEnum: 2
+};
+
+/**
+ * @constructor
+ */
+MyGame.Example.Test = function() {
+  /**
```

Comment:
@gwvo The comments could be used to add type-checking later on, e.g. with TypeScript.